cmd/kots/cli/get-joincommand.go

```diff
@@ -46,58 +47,78 @@ func GetJoinCmd() *cobra.Command {
 			if err != nil {
 				return err
 			}
-			fmt.Println(joinCmd)
-			return nil
+
+			if format == "string" {
+				fmt.Println(strings.Join(joinCmd, " "))
+				return nil
+			} else if format == "json" {
+				type joinCommandResponse struct {
+					Command []string `json:"command"`
+				}
+				joinCmdResponse := joinCommandResponse{
+					Command: joinCmd,
+				}
+				b, err := json.Marshal(joinCmdResponse)
+				if err != nil {
+					return fmt.Errorf("failed to marshal join command: %w", err)
+				}
+				fmt.Println(string(b))
+				return nil
+			}
+
+			return fmt.Errorf("invalid output format: %s", format)
 		},
 	}
 
+	cmd.Flags().StringVarP(&format, "output", "o", "string", "Output format. One of: string, json")
```

Comment:
sorry, but to be consistent with other commands, we should use `""` instead of `string`

Reply:
okay - so `""` or `string` gets string output, `json` gets json output?

Reply:
no need for `string` at all IMO

Reply:
updated again